commit 4346bc5815c635ecf5b738ea5e699f13b6c069c4
Author: Nikolaus Schulz <microschulz@web.de>
Date:   Sun Jul 10 00:18:05 2011 +0200

    manpage: minor grammar fix "allows to" -> "allows you to"
    
    This error was found by lintian.

diff --git a/archivemail.xml b/archivemail.xml
index 722b60e..9bc5642 100644
--- a/archivemail.xml
+++ b/archivemail.xml
@@ -570,7 +570,7 @@ Also, there exist broken servers which do not implement server side searches.
 (<citetitle>Internet Message Format</citetitle>) rules for the
 <token>local-part</token> of email addresses in mind.
 So, rather than enforcing an <acronym>URL</acronym>-style encoding of
-non-<acronym>ascii</acronym> and reserved characters, it allows to
+non-<acronym>ascii</acronym> and reserved characters, it allows you to
 double-quote the username and password.
 If your username or password contains the delimiter characters
 <quote>@</quote> or <quote>:</quote>, just quote it like this:
